tests/test_setup_command.py

```diff
@@ -108,3 +108,23 @@ def test_build_sphinx_return_nonzero_status(pkgroot, proc):
     print(out)
     print(err)
     assert proc.returncode != 0, 'expect non-zero status for setup.py'
+
```

Comment:
Please add more one blank line here.
PEP8 says two blank lines are needed before functions and classes on top level.